config/repos.yml

```diff
@@ -13,6 +13,21 @@ master:
   manageiq-ui-classic:
   manageiq-ui-service:
   manageiq_docs:
+fine:
+  manageiq:
+    has_rake_release: true
+  manageiq-appliance:
+  manageiq-appliance-build:
+  manageiq-content:
```

Comment:
This one here, needs the has_rake_release: true (which is present on latest master)